packages/debug/src/__tests__/debug.extended.test.ts

```diff
@@ -1,5 +1,14 @@
+import * as useDebugImport from '../index'
+
+let Debug: typeof useDebugImport.default
```

Comment:
Do we need to be global or can `importDebug` simply return it?

Reply:
Alternatively, i think `importDebug` can be called in `beforeEach`?

Reply:
I had to run a few iterations on that regard, but `importDebug` should be called after `beforeEach`, due to `process.env` shenanigans happening in the module scope, and each test mingling with `process.env` writes